statics/js/components/graph-layout.js

```diff
@@ -1192,6 +1192,35 @@ TopologyGraphLayout.prototype = {
     this.tick();
   },
 
+  styleReturn: function(d, values) {
+    if (d.active)
+      return values[0];
+    if (d.warning)
+      return values[1];
+    if (d.alert)
+      return values[2];
+    return values[3];
+  },
+
+  styleStrokeDasharray: function(d) {
+    return this.styleReturn(d, ["20", "20", "20", ""]);
+  },
+
+  styleStrokeDashoffset: function(d) {
+    return this.styleReturn(d, ["80 ", "80", "80", ""]);
+  },
+
+  styleAnimation: function(d) {
+    var animate =  function(speed) {
```

Comment:
sorry but can you fix the indentation of this line(" = ") and the following one (2 spaces only). It will avoid auto-reindent on other patches with linters. Otherwise that is OK for me